cmake/xcode.cmake

```diff
@@ -0,0 +1,44 @@
+cmake_minimum_required(VERSION 3.5)
+
+set(CTEST_PROJECT_NAME "Drafter")
+if("${CTEST_BUILD_NAME}" STREQUAL "")
+    set(CTEST_BUILD_NAME "${CTEST_BUILD_TYPE}")
+else()
+    set(CTEST_BUILD_NAME "${CTEST_BUILD_TYPE}/${CTEST_BUILD_NAME}")
+endif()
+
+set(CTEST_SOURCE_DIRECTORY ".")
+set(CTEST_BINARY_DIRECTORY "build")
+
+set(CTEST_CMAKE_GENERATOR "Xcode")
```

Comment:
@kylef I'd like to keep the OS X build environment though. So let's just use `unix.cmake` and run that on the `Xcode` image? Perhaps removing clang/linux builds?

Reply:
Yes that's what I meant. Building on macOS (using CircleCI xcode image) using `unix.cmake`, since that's usually what our users will be doing on their macOS platforms (installing via cmake as CLI). The homebrew package and our instructions are documenting that specific flow.